proxy-node-shared/src/test/java/uk/gov/ida/notification/shared/proxy/VspProxyTranslateResponseTest.java

```diff
@@ -109,7 +110,7 @@ public void shouldReturnTranslatedHubResponseAuthnFailed() {
         assertThat(VspScenario.AUTHENTICATION_FAILED).isEqualTo(response.getScenario());
         assertThat("123456").isEqualTo(response.getPid());
         assertThat(VspLevelOfAssurance.LEVEL_2).isEqualTo(response.getLevelOfAssurance());
-        assertThat(response.getAttributes()).isNull();
+        assertThat(response.getAttributes().orElse(null)).isNull();
```

Comment:
Should we be testing for `response.getAttributes().isEmpty()`?

Reply:
I suppose we should, although I think it's functionally the same.

Reply:
Yes - it's more clear to use the Optional's nicely-named methods